app/components/bali/tags/component.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+module Bali
+  module Tags
+    class Component < ApplicationViewComponent
+      renders_many :tag_items, ->(text: nil, href: nil, delete: false, **options) do
```

Comment:
Aqui podemos simplificar el nombre porque ya estamos dentro del **Tags** component. Es obvio que los items son tags también.

```suggestion
      renders_many :items, ->(text: nil, href: nil, delete: false, **options) do
```